consul_template/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+
+- name: Pull consul-template image
+  docker_image: name="{{ consul_template_image }}" tag="{{ consul_template_image_tag }}"
+  tags: ['consul_template', 'consul_template:install']
+
+- name: Copy systemd service file
+  template: >
+    src="{{ consul_template_service }}" dest="/etc/systemd/system/consul-template.service"
+    owner=root group=root
+  notify:
+    - reload systemd configuration
+    -  restart consul-template
```

Comment:
Whoops. Ansible didn't throw a syntax error, but it wouldn't surprise me if something like this becomes `rm -rf /`.